addons/l10n_syscohada/__manifest__.py

```diff
@@ -3,8 +3,6 @@
     'name': 'OHADA - Accounting',
     'category': 'Accounting/Localizations/Account Charts',
     'icon': '/l10n_syscohada/static/description/icon.jpeg',
-    'countries': ['bj', 'bf', 'cm', 'cf', 'km', 'cg', 'ci', 'ga', 'gn', 'gw', 'gq', 'ml', 'ne',
-                  'cd', 'sn', 'td', 'tg'],
```

Comment:
Removing this one is incorrect, as it breaks the automatic installation of this localization module.

Reply:
done